docs/application/dotnet/getting-started/wearable/first-app.md

```diff
@@ -64,45 +55,34 @@ To create a new Tizen .NET project:
 1.  Launch Visual Studio.
 2. In the Visual Studio menu, select **File &gt; New &gt; Project**.
 
-    ![Create a new project](media/cs_first_creating_menupath.png)
+    ![Create a new project](../media/cs_first_creating_menupath.png)
 
-    The New Project dialog opens.
+    A New Project window appears.
```

Comment:
I was referring to Microsoft style guide and that suggests, “appears" instead of "opens". 
However, opens is also correct.
If you recommend to use "opens". I will change it in my reviews. Please advise me.